openai.moderations/Module.md

```diff
@@ -0,0 +1,81 @@
+This is a generated connector for [OpenAI Moderations API](https://beta.openai.com/docs/api-reference/moderations) OpenAPI Specification. OpenAI is an American artificial intelligence research laboratory consisting of a non-profit corporation and a for-profit subsidiary. OpenAI conducts AI research with the declared intention of promoting and developing friendly AI. The OpenAI Moderations API provides a way to access new moderations models developed by OpenAI for content moderation tasks.
+
+## Prerequisites
+
+Before using this connector in your Ballerina application, complete the following:
+
+* Create an [OpenAI Account](https://beta.openai.com/signup/).
+* Obtain an API key by following [these instructions](https://platform.openai.com/docs/api-reference/authentication).
+
+## Quickstart
+
+To use the OpenAI Moderations connector in your Ballerina application, update the `.bal` file as follows:
```

Comment:
```suggestion
To use the OpenAI `moderations` connector in your Ballerina application, update the `.bal` file as follows:
```

For the name of the connector, keep the first letter in lowercase and have the name within backticks. Please change in all occurrences.